src/services/users/update-user.test.js

```diff
@@ -0,0 +1,68 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const updateUser = require('./update-user')
+
+
+describe('Update User', () => {
+	let user
+	let updatedInfo = {}
+
+	beforeEach(async () => {
+		user = await fixtures.insertUser()
+		updatedInfo = {
+			firstName: faker.name.firstName(),
+			lastName: faker.name.lastName(),
+			gender: faker.name.gender(),
+			jobRole: faker.name.jobTitle(),
+			department: faker.name.jobArea(),
+			address: faker.address.streetAddress(),
+			profilePictureUrl: faker.image.imageUrl()
+		}
+	})
+
+	it('should throw an error if user does not exist', async () => {
+		const { id } = user
+		const { firstName, lastName, gender } = updatedInfo
+		await db.query(
+			`DELETE FROM users
+             WHERE id = $1`, [id])
+
+		return expect(updateUser(id, { firstName, lastName,  gender }))
+			.to.be.rejectedWith(
+				'User not found')
+	})
+
+	it('should update user', async () => {
+		const { id } = user
+		const { firstName, lastName,
+			gender, jobRole, department, address, 
+			profilePictureUrl } = updatedInfo
+		const newUpdatedUser = await updateUser(
+			id, { firstName, lastName, 
+				gender, jobRole, department, address, 
+				profilePictureUrl })
+
+		const { rows } = await db.query(
+			`SELECT * FROM users
+             WHERE id = $1`, [id])
+
+		const  updatedUser = rows[0]
+
+		return expect(newUpdatedUser).to.eql({
+			id: updatedUser.id,
+			firstName: updatedUser.firstName,
```

Comment:
If we're checking if the user was updated, let's compare the `updatedUser` to the stuff generated in the `updatedInfo`

otherwise, it seems this test case will pass even if the `updateUser` function returns a user object without doing any update